workflow/templates/general_html_report/style.css

```
@@ -3,7 +3,7 @@
 }
 
 .center {
-  margin-left: 380px; /*this will center your wrapper*/
+  margin-left: 220px; /*this will center your wrapper*/
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)